app/src/main/java/edu/mills/cs115/AZCategoryFragment.java

```diff
@@ -1,11 +1,9 @@
 package edu.mills.cs115;
 
-import android.annotation.TargetApi;
 import android.database.Cursor;
 import android.database.sqlite.SQLiteDatabase;
 import android.database.sqlite.SQLiteException;
 import android.database.sqlite.SQLiteOpenHelper;
-import android.os.Build;
 import android.os.Bundle;
```

Comment:
Will you leave these lines blank in order to organize/categorize the import statements?